foundations/html_css/html-foundations/project-recipes.md

```diff
@@ -100,3 +100,17 @@ There are a couple of ways to go about doing this, but the simplest is this:
 -   Change the **Branch** from _none_ to _main branch_ and click **Save**. 
 -   It may take a few minutes (The GitHub website says up to 10, but we've seen it take up to an hour. Do not add a "theme" to your project, or you may have git conflicts, instead, be patient.) but your project should be accessible over the web from `your-github-username.github.io/your-github-repo-name` (obviously substituting your own details in the link).
 -   If your project does not publish after 1 hour, ensure that you have a file called `index.html` in the root of your repository and all the settings have been set correctly.  Go to your repo on GitHub and click on Actions, if there are no entries, then go back to the settings, change the **Branch** from _main branch_ to _none_ and click **Save**, then change the **Branch** from _none_ to _main branch_ and click **Save**.
+
+<div class="lesson-note" markdown="1">
+When you look at project submissions below, you might be asking:
+"Why do they look so good, should my project look like this?".
+
+Here is the answer:
+
+1- The main reason they look that good is that they are not done by beginners. They were likely submitted by people who had some experience with programming, or who have completed The Odin Project before and are coming back to make better websites.
+
+2- Your project should not look like that. Focus mainly on the requirements of the project.
```

Comment:
Suggesting removing the space and using lazy numbering here. It looks a lot cleaner in the preview.
```suggestion
1. The main reason they look that good is that they are not done by beginners. They were likely submitted by people who had some experience with programming, or who have completed The Odin Project before and are coming back to make better websites.
1. Your project should not look like that. Focus mainly on the requirements of the project.
```

Reply:
Ok, good idea, but your change uses "1." for both points instead of 1 and 2

Reply:
https://google.github.io/styleguide/docguide/style.html#use-lazy-numbering-for-long-lists

You should take the markdown for the whole page and put it in the [Markdown Preview Tool](https://www.theodinproject.com/lessons/preview), lemme know if what I have works.

Reply:
it works but there is a tiny change to make,
i made a comment that highlights it: [here](https://github.com/TheOdinProject/curriculum/pull/26223#issuecomment-1698033669)

Reply:
You think I am not understanding you. I understand what you are saying. You think I am erroneously putting two 1's instead of putting a 1 and a 2.

Using multiple `1.`s in a Markdown File Type is called Lazy Numbering. It's covered on the [link](https://google.github.io/styleguide/docguide/style.html#use-lazy-numbering-for-long-lists) I sent previously and in our [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md).

I am begging you, please take this code:
```md
1. Step one
1. Step two
```
And put it in the [Markdown Preview Too](https://www.theodinproject.com/lessons/preview)l. Please. We do this for a reason.

Reply:
okay, it was a misunderstanding, it wasn't clear to me what you asked.
I appreciate what you are doing and misunderstandings happen.